### PR TITLE
punto15

### PR DESCRIPTION
**Pull request by jeffrey20122021**

### CODE REVIEW

File: cypress/page/upload.page.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      cy.get(this.uploadBtn).click();
+    }
+  
+    public msgConfirmation(fileName: string, msg:string): void {
+        cy.get(this.msgconfirmUpload).should("have.text", msg);
+      cy.get(this.uploadedFile).should("contain", fileName);
+    }
+  }
+  
+  export {UploadPage};
```
Comment (holgiosalos) [inline cypress/page/upload.page.ts:31]: EOF

File: cypress/page/upload.page.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }
+  
+    public uploadFile(file: string): void {
+      cy.get(this.selectFile).attachFile(file);
+      cy.get(this.uploadBtn).click();
+    }
+  
+    public msgConfirmation(fileName: string, msg:string): void {
+        cy.get(this.msgconfirmUpload).should("have.text", msg);
+      cy.get(this.uploadedFile).should("contain", fileName);
```
Comment (holgiosalos) [inline cypress/page/upload.page.ts:27]: fix indentation

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -29,3 +29,6 @@ Cypress.on("uncaught:exception", (err, runnable) => {
     // failing the test
     return false;
 });
+
+import 'cypress-file-upload';
+
```
Comment (holgiosalos) [inline cypress/support/commands.ts:34]: import should go at before the Cypress.on call

**Review by holgiosalos — COMMENTED** [review]
Remember to run the lint:fix command before sending a PR to avoid fixable issues appear on the CI

File: cypress/support/commands.ts (hunk 3 above)
Comment (jeffrey20122021) [reply cypress/support/commands.ts]: all done @holgiosalos

File: cypress/page/upload.page.ts (hunk 2 above)
Comment (holgiosalos) [reply cypress/page/upload.page.ts:27]: lines 26 and 27 needs to be aligned, fix indentation

File: cypress/page/upload.page.ts (hunk 1 above)
Comment (holgiosalos) [reply cypress/page/upload.page.ts:31]: still see the EOF warning
image

File: cypress/support/commands.ts [hunk 4]
```diff
@@ -24,8 +24,12 @@
...
 // Ignoring uncaught exceptions since errors from external apps should not stop de workshop
+import 'cypress-file-upload';
 Cypress.on("uncaught:exception", (err, runnable) => {
     // returning false here prevents Cypress from
     // failing the test
     return false;
 });
+
+
+
```
Comment (holgiosalos) [inline cypress/support/commands.ts:35]: remove the extra blank lines at the end of the file, it is just needed one blank line

Comment (jeffrey20122021) [reply cypress/support/commands.ts:35]: all done @holgiosalos

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**